doto-frontend/src/components/pages/Calendar/Calendar.js

```diff
@@ -53,6 +84,24 @@ const Calendar = () => {
         setOpen(false);
     };
 
+    const handleStatsOpen = () => {
+        if (statsOpen) {
+            setStatsOpen(false);
+        } else {
+            var medTasks = tasks.filter(function(task) {
+                return task.priority === 20 && task.isComplete;
+            });
+            var lowTasks = tasks.filter(function(task) {
+                return task.priority === 30 && task.isComplete;
+            });
+            var highTasks = tasks.filter(function(task) {
+                return task.priority === 10 && task.isComplete;
+            });
+            setPriorityStats([highTasks.length, medTasks.length, lowTasks.length]);
```

Comment:
Some tasks can have no priority, but they're not shown in this count. Which can be confusing as a user might complete a task (that has no priority) but it won't update their stats. Not blocking, as it's not essential.